index.html

```diff
@@ -1,12 +1,18 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
   <head>
     <meta charset="UTF-8" />
-    <title>Web QR ZXing</title>
+    <title>WebQR ZXing</title>
     <meta
       name="viewport"
       content="width=device-width, height=device-height, initial-scale=1.0"
     />
+    <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+    <link
+      rel="manifest"
+      href="/manifest.webmanifest"
+      crossorigin="use-credentials"
+    />
```

Comment:
`crossorigin="use-credentials"` is needed for vercel preview
https://stackoverflow.com/questions/60669830/what-is-the-main-use-of-the-site-webmanifest-file